src/browser_app_skeleton/webpack.mix.js

```diff
@@ -6,15 +6,35 @@
  |
  | Docs: https://github.com/JeffreyWay/laravel-mix/tree/master/docs#readme
  */
-
 let mix = require("laravel-mix");
+let plugins = [];
 
 // Customize the notifier to be less noisy
 let WebpackNotifierPlugin = require('webpack-notifier');
 let webpackNotifier = new WebpackNotifierPlugin({
   alwaysNotify: false,
   skipFirstNotification: true
 })
+plugins.push(webpackNotifier)
+
+// Compress static assets in production
+if (mix.inProduction()) {
+  let CompressionWepackPlugin = require('compression-webpack-plugin');
+  let gzipCompression = new CompressionWepackPlugin({
+    compressionOptions: { level: 9 },
+    test: /\.js$|\.css$|\.html$|\.svg$/
+  })
+  plugins.push(gzipCompression)
+  // add additional compression plugins here. For example
+  // if you want to add brotli compression...
+  // let brotliCompression = new CompressionWepackPlugin({
```

Comment:
Could you add an empty line above this so it is clear where the code starts?

Reply:
I think this is what you mean...

Reply:
Yep looks good!